src/index.ts

```diff
@@ -1,7 +1,14 @@
 // @ts-expect-error
-import { redirects } from "../redirects.yml";
+import { redirects, prefix } from "../redirects.yml";
 
-const redirectMap: Map<string, string> = new Map(Object.entries(redirects));
+const redirectPrefix: string = prefix || "/r/";
```

Comment:
```suggestion
const redirectPrefix: string = prefix || "/r";
```

Reply:
If it's a prefix, it should, by convention, not include a trailing slash. This way users could define a prefix like

```yaml
prefix: /rawr
redirects:
  something: https://yay.com
```

Otherwise, LGTM! Thanks

Reply:
good point! i fix